gnd/src/main/java/com/google/android/gnd/ui/home/mapcontainer/MapContainerFragment.java

```diff
@@ -107,7 +107,7 @@ public void onCreate(@Nullable Bundle savedInstanceState) {
         .toObservable()
         .flatMap(MapAdapter::getTileProviders)
         .as(disposeOnDestroy(this))
-        .subscribe(mapContainerViewModel::onTileProvider);
+        .subscribe(mapContainerViewModel::queueTileProvider);
```

Comment:
Not sure `queue` is the right name here, since it implies an actual queue is being used, and not a stream. In this case "add" would be correct, since it's really a deferred add operation, but this can be changed later if we like.